src/components/App.js

```diff
@@ -0,0 +1,66 @@
+import "../styles/App.css";
+import React, { useEffect, useState } from "react";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import getForecast from "../requests/getForecast";
+import SearchForm from "./SearchForm";
+
+function App() {
```

Comment:
Is there a reason why all the components are declared using non ES6 syntax (as opposed to functional definitions like `const App = () => { ... }`)?

Reply:
I did look at swapping them to ES6 but got ESlint errors when doing so with the following error message: 'Function component is not a function declaration'. I couldn't figure out how to fix it. So presumed i was supposed to use old school functions

Reply:
Any ideas how to fix this - just been having a look - but struggling to fix this. I've changed all functions to ES6
![Screenshot from 2022-09-24 12-08-06](https://user-images.githubusercontent.com/104670473/192094626-ed366513-a2ef-4f84-a73b-16d4b2236d59.png)

Reply:
This is because eslint is enforcing a particular style of function declaration. You can choose whatever style you want to use, options are given in the docs [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md)
In this case, if you want to use functions in the format 
```
const App = () => {
  // your code here
}
```
add this block to the rules object of your `.eslintrc.json`:
```
"react/function-component-definition": [
  2,
  { "namedComponents": "arrow-function" }
]
```

Then it _only_ allow components to be defined using that syntax and keep things uniform

Reply:
Ah yes, that makes sense. Thanks so much for all your feedback Stu - really useful.